server/app/views/BaseHtmlView.java

```diff
@@ -79,6 +81,22 @@ protected static ButtonTag makeSvgTextButton(String buttonText, Icons icon) {
             span(buttonText).withClass(Styles.TEXT_LEFT));
   }
 
+  protected static SpanTag space() {
+    return span(" ");
```

Comment:
Is `" "` ok in this case or should it be a `&nbsp;`

Reply:
Well it is true, that is under the assumption of latin-based languages. I guess there at most 4 languages civiform supports / will support that might not have the same character for spaces....

Maybe I should put the space in the messages file instead?

Reply:
Done

Reply:
Done. Hows this now?

Reply:
I'm not sure about how other languages would use different space characters. In general I'd just say keep it as part of the full text being translated rather than having a separate character as it's own thing.

Reply:
yeah, back then I didn't know how to put a space at the end of the message entry, which is why I did that I think...